content/videos/challenges/34-diffusion-lmited-aggregation/showcase/contribution1.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "DLA Simulator 2.0",
+  "url": "https://github.com/infernal9999/Diffusion-limited-Aggregation",
+  "author": {
+      "name": " by Diptanu Roy",
```

Comment:
Is the author name supposed to be just "Diptanu Roy"?

Reply:
I can fix that, sorry I missed it.